### PR TITLE
fix: fix the button disable style

### PR DESCRIPTION
**Pull request by haishengwu-okta**
OKTA-260287
demo

### CODE REVIEW

File: src/v2/controllers/FormController.js [hunk 1]
```diff
@@ -77,13 +77,29 @@ export default BaseLoginController.extend({
...
+   * SignIn widget has its own (hacky) way to customize the button disabled state:
+   * adding `link-button-disabled` despite the name was intend only to disable
+   * `link-button`.
+   * Instead of doing decent refactor, we want to follow the convention for now.
+   *
+   * @param {boolean} disabled whether add extra disable CSS class.
+   */
+  toggleFormButtonState: function (disabled) {
+    var button = this.$el.find('.o-form-button-bar .button');
+    button.toggleClass('link-button-disabled', disabled);
```
Comment (nikhilvenkatraman-okta) [inline src/v2/controllers/FormController.js]: was this class not getting added to this button ?

File: assets/sass/okta-theme.scss [hunk 2]
```diff
@@ -52,19 +52,6 @@
     text-decoration: none;
     cursor: pointer;
   }
-
```
Comment (nikhilvenkatraman-okta) [inline assets/sass/okta-theme.scss]: I am guessing this has been here for a while ? since we release 3.0 ?

Comment (haishengwu-okta) [reply assets/sass/okta-theme.scss]: this was added in the OIE commit.

File: src/v2/controllers/FormController.js (hunk 1 above)
Comment (haishengwu-okta) [reply src/v2/controllers/FormController.js]: Courage doesn't add `link-button-disabled` for `input` buttons.

**Pull request merged**